packages/schematics/angular/migrations/update-14/update-angular-config.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import { Rule } from '@angular-devkit/schematics';
+import { allTargetOptions, updateWorkspace } from '../../utility/workspace';
+
+/** Migration to remove deprecated/removed builder options. */
```

Comment:
Do we expect to remove more options in this schematic? This feels more vague than it needs to be if we're just dropping `showCircularDependencies`, but maybe you have bigger plans for it?

Reply:
I though to name it generic in case we need to remove addition options. 

I checked again and it seems we don’t have any more deprecations that we can remove in v14. 

Therefore , I’ll rename it to be more explicit.